lib/publiccloud/provider.pm

```diff
@@ -437,7 +442,12 @@ sub terraform_destroy {
     }
     my $ret = script_run('terraform destroy -no-color -auto-approve', get_var('TERRAFORM_TIMEOUT', TERRAFORM_TIMEOUT));
     unless (defined $ret) {
-        type_string(qq(\c\\));        # Send QUIT signal
+        if (is_serial_terminal()) {
+            type_string(qq(\c\\));    # Send QUIT signal
+        }
+        else {
+            send_key('ctrl-\\');      # Send QUIT signal
```

Comment:
Why not `ctrl-c`?
```suggestion
            send_key('ctrl-c');      # Send QUIT signal
```

Reply:
Very good question, but it would be for the original creator of this module. It was also new to me, seems like `ctrl+\` also creates a core dump? https://en.wikipedia.org/wiki/Control-%5C

Anyhow my intent was only to fix the bug instead of changing any logic or decision made before or refactor something.

I see three options:
- stick to the bug fix with minimum change
- include some refactoring/adapting
- create a new poo issue to tackle that separately

what do you think?

Reply:
`ctrl-c` is quite common to catch and then the programm still go on... So I send QUIT to avoid such situation.

Reply:
I didn't know that. Than you